src/main/webapp/index.html

```diff
@@ -70,4 +61,7 @@
       </div>
     </nav>
   </body>
+  
+  <!-- JavaScript -->
+  <script src="script.js"></script>
```

Comment:
Any specific reason to have the link to JS file here?

Reply:
Already explained in last PR :)